integreat_cms/cms/views/push_notifications/push_notification_sender.py

```diff
@@ -87,17 +87,30 @@ def send_pn(self, pnt):
         :rtype: ~requests.Response
         """
         if settings.DEBUG:
-            region_slug = Region.objects.get(
-                id=settings.TEST_BLOG_ID
-            ).slug  # Testumgebung - prevent sending PNs to actual users in development
+            region = Region.objects.get(
+                slug="testumgebung"
+            )  # Testumgebung - prevent sending PNs to actual users in development
         else:
-            region_slug = self.push_notification.region.slug
+            region = self.push_notification.region
         payload = {
-            "to": f"/topics/{region_slug}-{pnt.language.slug}-{self.push_notification.channel}",
+            "to": f"/topics/{region.slug}-{pnt.language.slug}-{self.push_notification.channel.lower()}",
```

Comment:
Instead of `.lower()`, we could also make sure the value in the settings file is in lower case:
```
CHANNELS = (("news", "News"),) 
```
instead of:

https://github.com/digitalfabrik/integreat-cms/blob/f708ba12b2fa49163cc284ba0fd250e7f9736219/integreat_cms/core/settings.py#L702

Reply:
And since the tab is called "Nachrichten" on german, we could also think about translating the second "News" tuple value...

Reply:
Done.